custom_components/openplantbook/manifest.json

```diff
@@ -6,6 +6,8 @@
   "documentation": "https://github.com/Olen/home-assistant-openplantbook/",
   "iot_class": "cloud_polling",
   "issue_tracker": "https://github.com/Olen/home-assistant-openplantbook/issues",
-  "requirements": ["pyopenplantbook==0.0.4"],
-  "version": "1.2.0"
+  "dependencies": ["recorder","history"],
+  "requirements": ["openplantbook-sdk==0.4.5","json-timeseries==0.1.4"],
+  "version": "1.3.0",
```

Comment:
Please keep the beta version tag for now.  Github actions uses this to auto create releases.

Reply:
Got it. Done in the latest commit.